src/index.css

```diff
@@ -510,4 +510,61 @@ Notes on possible css class implementation:
 	padding-left: 0.75rem;
 }
 
-/** DateTimePicker **/
+/** Items **/
```

Comment:
Consider creating a separate css file which can be imported in elements styleImports

Reply:
Agree